src/dialogs/detectorDialog.tsx

```diff
@@ -60,8 +59,7 @@ export default function DetectorDialog(props: {
             <Grid item xs={12} sm={12} md={12} lg={7}>
               <DetectorTable />
             </Grid>
-            <Grid item md={0} lg={1} >
-            </Grid>
+            <Grid item md={0} lg={1}></Grid>
```

Comment:
why leave an empty grid element here? whitespace can be achieved with `margin` css properties